Reflection_Engine/Query/OutputAttributes.cs

```diff
@@ -54,8 +54,8 @@ public static List<OutputAttribute> OutputAttributes(this MethodBase method)
                         if (types[i] != null)
                         {
                             desc += Environment.NewLine;
-                            QuantityAttribute quantityAttribute = outputDefs[i].Quantity;
-                            desc += types[i].UnderlyingType().Type.Description(quantityAttribute);
+                            InputClassificationAttribute clasification = outputDefs[i].Classification;
```

Comment:
Should be `classification` for the variable name - 2 `s` :smile:

Reply:
Fixed